randovania/games/planets_zebeth/gui/preset_settings/planets_zebeth_patches_tab.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from PySide6 import QtWidgets
+
+from randovania.games.planets_zebeth.layout import PlanetsZebethConfiguration
+from randovania.gui.preset_settings.preset_tab import PresetTab
+
+if TYPE_CHECKING:
+    from randovania.game_description.game_description import GameDescription
+    from randovania.gui.lib.window_manager import WindowManager
+    from randovania.interface_common.preset_editor import PresetEditor
+    from randovania.layout.preset import Preset
+
+
+class PresetPlanetsZebethPatches(PresetTab):
```

Comment:
not sure if you have any planned patches yet, but may or may not wanna add a TODO for here